server/socketEvents.js

```diff
@@ -0,0 +1,41 @@
+const Message = require('./models/Message');
+
+exports = module.exports = function (io) {
+    // Set socket.io listeners.
+    io.on('connection', (socket) => {
+        console.log('a user connected: ' + socket.id);
+
+        socket.on('new message', (msgObject) => {
+
+            const reply = new Message({
+                conversationId: msgObject.conversationId,
+                body: msgObject.msg,
+                author: msgObject.userid
+            });
+
+            reply.save((err, sentReply) => {
+                if (err) {
+                    return next(err);
```

Comment:
I don't think there's a `next()` since this isn't middleware, maybe just do a `console.log()`?